client/consensus/babe/src/lib.rs

```diff
@@ -1461,7 +1461,7 @@ where
 		// this way we can revert it if there's any error
 		let mut old_epoch_changes = None;
 
-		// Use an extra scope to make the compiler happy, because otherwise he complains about the
+		// Use an extra scope to make the compiler happy, because otherwise it complains about the
```

Comment:
@crystalin

> Using "he" in this sentence would imply that the subject of the sentence is a person who identifies as male. However, the subject is a compiler, which is not a person and does not have a gender. By using "it," the sentence is neutral and does not make any assumptions about the subject's gender identity. Additionally, using "it" makes the language more inclusive and respectful to individuals who do not identify as binary.

I've double-checked the other uses of "it" elsewhere and replaced "it" with "they" since there is an attached identity as the "depositor" and "validator".